packages/flutter/lib/src/cupertino/text_form_field_row.dart

```diff
@@ -244,6 +246,11 @@ class CupertinoTextFormFieldRow extends FormField<String> {
   /// nature of the input.
   final Widget? prefix;
 
+  /// A widget that is displayed at the end of the row.
```

Comment:
Both of these paragraphs seem like they are saying the same thing. Maybe replace with `An optional [Widget] that is displayed at the end of the row.`

Reply:
Sure

Reply:
Done 👍✅